internal/commands/cnab.go

```diff
@@ -157,7 +157,7 @@ func prepareCredentialSet(b *bundle.Bundle, opts ...credentialSetOpt) (map[strin
 	_, requiresDockerContext := b.Credentials[internal.CredentialDockerContextName]
 	_, hasDockerContext := creds[internal.CredentialDockerContextName]
 	if requiresDockerContext && !hasDockerContext {
-		return nil, errors.New("no target context specified. Use --target-context= or DOCKER_TARGET_CONTEXT= to define it")
+		return nil, errors.New("no target context specified. Use DOCKER_TARGET_CONTEXT= to define it")
```

Comment:
The error still mentions "target context specified"

Reply:
Do we really still want to advertise about this `DOCKER_TARGET_CONTEXT` feature?